app/src/main/java/com/example/lastfm/ArtistInfo.java

```diff
@@ -0,0 +1,416 @@
+
+package com.example.lastfm;
+
+import java.util.List;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+class ArtistData {
```

Comment:
split all these classes on different files

Reply:
Splited.

Reply:
unit all these classes in one package

Reply:
Created package and moved all these classes